eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/configuration/ImportDbsnpVariantsJobConfiguration.java

```diff
@@ -28,6 +28,7 @@
 
 import static uk.ac.ebi.eva.accession.dbsnp.configuration.BeanNames.IMPORT_DBSNP_VARIANTS_JOB;
 import static uk.ac.ebi.eva.accession.dbsnp.configuration.BeanNames.IMPORT_DBSNP_VARIANTS_STEP;
+import static uk.ac.ebi.eva.accession.dbsnp.configuration.BeanNames.VALIDATE_CONTIGS_STEP;
 
 @Configuration
 @EnableBatchProcessing
```

Comment:
Now that we have 2 steps, we should document this job workflow (see [example](https://github.com/EBIvariation/eva-pipeline/blob/master/src/main/java/uk/ac/ebi/eva/pipeline/configuration/jobs/GenotypedVcfJobConfiguration.java#L46-L54) from eva-pipeline).

Reply:
IMO, unless you want to go to the detail level of processors, I wouldn't do that right now. the job is literally 2 sequential simple steps with they readers, processors, writers.

Maybe when we implement the flag to skip the first step it is worth it, but not now.

Reply:
Sounds fair.